### PR TITLE
PHX-229 Turn off toggle for map overlay that is not visible

### PR DESCRIPTION
**Pull request by billli0**
### Issue #:
PHX-229

### CODE REVIEW

File: packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js [hunk 1]
```diff
@@ -91,12 +91,6 @@ export const TitleAndDatePickerComponent = ({
   setPinnedOverlay,
   maxSelectedOverlays,
 }) => {
-  const [isSwitchedOn, setIsSwitchedOn] = useState(true);
-
```
Comment (billli0) [inline packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js:95]: Didn't notice `isSwitchedOn` is overlapping with `displayedMapOverlays`. Need to remove.

**Review by kael89 — APPROVED** [review]
Fix looks good, thanks @billli0 !

**Comment by billli0** [comment]
@kael89 Not a fix but a new requirement. But I take the approval as it is!

**Comment by kael89** [comment]
> @kael89 Not a fix but a new requirement. But I take the approval as it is!

Oops I didn't realise that, thanks for clarifying! 🙂

**Comment by billli0** [comment]
@kael89 Latest commit to make `maxSelectedOverlays` globally to fix the failed test. Wanna take a quick review?

**Review by kael89 — APPROVED** [review]
Fix looks good @billli0 , thanks!

**Pull request merged**